src/Modules/Visualization/ShowField.cc

```diff
@@ -409,8 +365,8 @@ void ShowFieldModule::renderFacesLinear(
   // Attempt some form of precalculation of iboBuffer and vboBuffer size.
   // This Initial size estimation will be off quite a bit. Each face will
   // have up to 3 nodes associated.
-  uint32_t iboSize = mesh->num_faces() * sizeof(uint32_t) * 3;
-  uint32_t vboSize = mesh->num_faces() * sizeof(float) * 3;
+  uint32_t iboSize = static_cast<uint32_t>(mesh->num_faces() * sizeof(uint32_t) * 3);
+  uint32_t vboSize = static_cast<uint32_t>(mesh->num_faces() * sizeof(float) * 7);
```

Comment:
Same here, name the constants 3 and 7. The 7 was a 3, reader has no idea why it changed.